app/code/Magento/Customer/Ui/Component/Listing/AttributeRepository.php

```diff
@@ -156,10 +156,10 @@ protected function getOptionArray(array $options)
      * Return customer group's metadata by given group code
      *
      * @param string $code
-     * @return []
+     * @return string null
```

Comment:
Looks like return type should be array|null. Am I wrong?

Reply:
I change to return array and null